src/components/HoverCard/HoverCardOnUsername.tsx

```diff
@@ -89,7 +89,9 @@ export default function HoverCardOnUsername({ userRef, queryRef }: Props) {
   return (
     <StyledContainer onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
       <StyledLinkContainer>
-        <InteractiveLink to={`/${user.username}`}>{user.username}</InteractiveLink>
+        <Link href={`/${user.username}`}>
+          <TitleDiatypeM>{user.username}</TitleDiatypeM>
+        </Link>
```

Comment:
jess changed up how usernames get rendered in the feed event header 
<img width="241" alt="image" src="https://user-images.githubusercontent.com/6754223/195634409-41a1a59c-747c-4050-9ba0-401d20104935.png">